# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523298211 25200
#      Mon Apr 09 11:23:31 2018 -0700
# Branch stable
# Node ID f04a5543afda2c2a345726a4ba3c2baa3cfab9fd
# Parent  27fe7e6aba317874f48d5d7e09815e724b01c257
Fix typo in xml of Dutch translation in cset 27fe7e6aba31.

* libgui/languages/nl_NL.ts: Remove 'type =' fragment.

diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -2756,19 +2756,20 @@ Selecteer s.v.p. een andere of annuleer 
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
 with the selected encoding %1.
 Using it will result in data loss!
 
 Do you want to choose another encoding?</source>
-        <translation type=>De huidige inhoud van de editor kan niet worden gecodeerd
+        <translation>De huidige inhoud van de editor kan niet worden gecodeerd
 met de geselecteerde codering %1.
 Deze toch gebruiken leidt tot gegevensverlies!
+
 Wilt u een andere codering kiezen?</translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Het lijkt erop dat &apos;%1&apos; is gewijzigd door een ander programma. Wil je het opnieuw laden?</translation>
     </message>
     <message>
